modules/nw-networkpolicy-optimize-ovn.adoc

```diff
@@ -104,11 +104,11 @@ spec:
 You can apply this optimization when only multiple selectors are expressed as one. In cases where selectors are based on different labels, it may not be possible to apply this optimization. In those cases, consider applying some new labels for network policy optimization specifically.
 
 [id="nw-networkpolicy-external-ip-ovn_{context}"]
-== NetworkPolicies and external IPs in OVN-Kubernetes
+== NetworkPolicy and external IPs in OVN-Kubernetes
```

Comment:
Because we say "ClusterUserDefinedNetwork CR. . ." in those titles, maybe we should always put "CR" in the title to indicate the custom resource. 

```suggestion
== NetworkPolicy CR and external IPs in OVN-Kubernetes
```